lib/get-fail-comment.js

```diff
@@ -1,7 +1,7 @@
 const HOME_URL = "https://github.com/semantic-release/semantic-release";
 const FAQ_URL = `${HOME_URL}/blob/caribou/docs/support/FAQ.md`;
 const GET_HELP_URL = `${HOME_URL}#get-help`;
-const USAGE_DOC_URL = `${HOME_URL}/blob/caribou/docs/usage/README.md`;
+const USAGE_DOC_URL = `${HOME_URL}/blob/master/docs/support/FAQ.md`;
```

Comment:
```suggestion
const USAGE_DOC_URL = `${HOME_URL}/blob/master/docs/usage/README.md`;
```

this one should still point to usage, but should get the branch updated like you did